timeline/templates/timeline/side_timeline.html

```diff
@@ -6,8 +6,7 @@
 {% load projectroles_common_tags %}
```

Comment:
The file name of this template is not consistent with SODAR Core conventions. It should remain `timeline_site.html`.